helper-npm-update

```diff
@@ -8,10 +8,12 @@
 #
 # --dev - Also update devDependencies
 #
-# --no-package-lock - Don't write a package-lock.json as
-#   we don't use it and it can cause issues with `npm ls`,
-#   which is run by subsequent tasks in CI:
-#   https://github.com/npm/npm/issues/19393
+# --no-package-lock - While we now use package-lock.json 
+#   in our repos, skip package-lock.json in this instance 
+#   as we're bypassing the normal npm install process     
+#   and don't want to accidentally modify the file in CI.
+#   Normal dependencies are still installed via `make install` 
+#   which does use the package-lock.
```

Comment:
I think this comment might need to be amended slightly to reflect that the `npm install` is not run at all in this command (but [`npm update`](https://docs.npmjs.com/cli/v7/commands/npm-update) is).